ganttproject/src/main/java/net/sourceforge/ganttproject/storage/Xlog.kt

```diff
@@ -29,6 +29,40 @@ data class InitRecord(
   val projectXml: String
 )
 
+@Serializable
+sealed class OperationDto() {
+  @Serializable
+  data class InsertOperationDto(
+    val tableName: String,
+    val values: Map<String, String>
+  ): OperationDto()
+
+  @Serializable
+  data class UpdateOperationDto(
```

Comment:
I think we need a condition for update as well. We use task uid for that.

Reply:
Add the processing of these conditions in ColloboqueServer::applyOperation ?

Reply:
Yup, forgot to include file in commit